pkg/controller/nodemaintenance/nodemaintenance_controller_test.go

```diff
@@ -106,6 +106,23 @@ var _ = Describe("updateCondition", func() {
 			Expect(maintanance.Status.EvictionPods).To(Equal(2))
 			Expect(maintanance.Status.TotalPods).To(Equal(2))
 		})
+		It("owner ref should be set properly", func() {
+			r.initMaintenanceStatus(nm)
+			maintanance := &nodemaintenanceapi.NodeMaintenance{}
+			err := cl.Get(context.TODO(), req.NamespacedName, maintanance)
+			node, err := cs.CoreV1().Nodes().Get("node01", metav1.GetOptions{})
```

Comment:
I see it is done at other places as well, but you should not hardcode node names.
The test can and will run on other CIs than KubeVirtCI with differenent node names.
I leave it up to you if you change it in this PR (at all places please) or in a follow up.

Reply:
this is a unit test, it doesn't run on any CI, in any event this is not related to any real nodes.

Reply:
oh yes, of course, sorry :slightly_smiling_face: